client/state/geolocation/schema.js

```diff
@@ -0,0 +1,12 @@
+export const geolocationSchema = {
+	type: 'object',
+	properties: {
+		latitude: { type: 'string' },
+		longitude: { type: 'string' },
+		countryShort: { type: 'string' },
+		countryLong: { type: 'string' },
+		region: { type: 'string' },
+		city: { type: 'string' }
```

Comment:
So the schema is going to be used against what we store in the Redux tree, vs what comes back in a response. So this will always throw out stored data.

Our action is crafted like:
![screen shot 2016-09-02 at 4 03 31 pm](https://cloud.githubusercontent.com/assets/1270189/18220453/f6e9d3da-7126-11e6-89ba-145a0114d02d.png)

But the reducer is storing:
![screen shot 2016-09-02 at 4 03 50 pm](https://cloud.githubusercontent.com/assets/1270189/18220454/07a84a80-7127-11e6-861a-6243d19f3344.png)